distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/restito/ChunkedContent.java

```diff
@@ -81,57 +73,97 @@ public static Action chunkedContent(String responseMessage, Duration messageDela
      * @return composite action that holds the headers required for a plain text response as well
```

Comment:
I added a `headerCapture` to the params. Even though it's private, it's a ctor so I feels weird having no javadocs.